post-checkout.rb

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env ruby
+
+heads = '.git/refs/heads/'
+issues = '.git/refs/issues/'
+
+Dir.new(heads).each do |f|
+  next if f.match? /^\.+$/ # skipping '.' and '..'
```

Comment:
FYI https://blog.bigbinary.com/2017/11/21/ruby-2_5-introduces-dir-children-and-dir-each_child.html